safekeeper/src/http/routes.rs

```diff
@@ -464,11 +504,18 @@ pub fn make_router(conf: SafeKeeperConf) -> RouterBuilder<hyper::Body, ApiError>
             "/v1/tenant/:tenant_id/timeline/:timeline_id/file/:filename",
             |r| request_span(r, timeline_files_handler),
         )
+        .post(
+            "/v1/tenant/:tenant_id/timeline/:source_timeline_id/copy",
+            |r| request_span(r, timeline_copy_handler),
+        )
         // for tests
         .post("/v1/record_safekeeper_info/:tenant_id/:timeline_id", |r| {
             request_span(r, record_safekeeper_info)
         })
         .get("/v1/debug_dump", |r| request_span(r, dump_debug_handler))
+        .post("/v1/tenant/:tenant_id/timeline/:timeline_id/digest", |r| {
```

Comment:
this should be GET, no?

Reply:
I wonder if we should also mention it in the yml API description. It's only needed for testing but it might make sense for documentation purposes?

Reply:
I'll made it POST because arguments are easy to pass in JSON body (and I had bad experience with writing `curl http://localhost.../debug_dump?all=true&disk=false`, try to guess what's wrong in this curl 🥲 ) 

But it makes sense to use GET when we're not modyfing the state, I'll change it to GET.

Reply:
it runs curl in background due to missing quotes?)

Reply:
Changed to GET